### ISSUE TITLE
quay.io/coreos/etcd and coreos.com/coreos/etcd behave differently

### ISSUE THREAD

**Issue by steveeJ** [coreos/etcd#6057]
The two container images seem to have different mounts and data directory paths. It'd be helpful to align the behavior of both.

**Comment by xiang90** [comment]
@steveeJ No matter which one we changes, we will introduce backward incompatibility issue? Users use new container image cannot start the previous etcd by default due to the data dir change?

**Comment by steveeJ** [comment]
@xiang90 would you be OK using symlinks for compatibility?

**Comment by xiang90** [comment]
@steveeJ Yes. That works for me. But we also need a depreciation path. But it would be a long term goal.

**Comment by xiang90** [comment]
@steveeJ Do you have time to get this fixed?
